osquery/remote/http_client.cpp

```diff
@@ -203,21 +203,27 @@ void Client::createConnection() {
 
 void Client::encryptConnection() {
   boost::asio::ssl::context ctx{boost::asio::ssl::context::sslv23};
+  const auto remote_hostname = *client_options_.remote_hostname_;
```

Comment:
Does this need to be declared so high up? I'm not seeing anything making modifications to `client_options_.remote_hostname_` or using `remote_hostname` until we check it in the callback. Could we remove this and just pass `*client_options_.remote_hostname_` directly in the callback?

Reply:
It's also used on line 250, but the main purpose I think was just to shorten the name.
I'll change it!